migrations/versions/4d93784a25e5_add_authz_column.py

```diff
@@ -38,8 +38,10 @@ def upgrade():
 
     # extract existing PK (guid) and authz data (resource_path) from the metadata column
     connection = op.get_bind()
-    results = connection.execute("SELECT guid, data from metadata").fetchall()
-    for r in results:
+    query = "SELECT guid, data from metadata"
+    results = connection.execute(query)
+    r = results.fetchone()
```

Comment:
querying the records one by one might be a bit slow? we should be able to query a few at a time without impacting the memory usage negatively